README.md

```diff
@@ -1600,12 +1604,15 @@ Long-term capital gains tax is often lower than ordinary income tax.
 Many investors hold assets for longer than a year in order to qualify for the lesser tax
 burden of long-term capital gains.
 
-🄳 **Alternative minimum tax (AMT)** is an
-[entirely separate kind of tax](https://en.wikipedia.org/wiki/Alternative_minimum_tax) that is
-neither ordinary income tax nor employment tax, and has its own rules and only applies in
-some situations.
-This type of tax does not come up for many people, but higher income earners and people in
-special situations often have to pay very large AMT bills.
+🄳 **Alternative minimum tax (AMT)** is a
+[supplemental income tax](https://en.wikipedia.org/wiki/Alternative_minimum_tax) that applies
+to certain individuals in some situations.
+This type of tax does not come up for many taxpayers, but higher income earners and people
+in special situations may have to pay large AMT bills.
+AMT was first enacted in [1979](https://www.taxpolicycenter.org/briefing-book/what-amt) in
+response to reports that 155 wealthy individuals had paid no income tax in 1966. It is
+[calculated differently](https://www.nerdwallet.com/blog/taxes/alternative-minimum-tax-amt/)
+than the standard income tax.
```

Comment:
Pls preserve the original distinctions from ordinary income tax and employment tax (terms already defined).

It is not the same as ordinary income tax or employment tax, and is
calculated [according to its own rules](https://www.nerdwallet.com/blog/taxes/alternative-minimum-tax-amt/).